### ISSUE TITLE
Add accessibility testing

### ISSUE THREAD

**Issue by gemfarmer** [18F/18f.gsa.gov#1906]
This repo currently lacks accessibility testing!

An automatic tester, such as [pa11y](https://github.com/pa11y/pa11y) or even fancier testing CI like [continu11y](https://github.com/18F/continua11y) would be a great addition.

**Comment by coreycaitlin** [comment]
I think this is also discussed in #1896 and resolved with #1924
